### ISSUE TITLE
unreachable 

### ISSUE THREAD

**Issue by ktye** [mathetake/gasm#13]
Running this module, triggers _unreachable_ opcode 0x00 where it shouldn't.
It happens in the inner _if_ block.
```
(module
  (type (;0;) (func (param i32) (result i32)))
  (func (;0;) (type 0) (param i32) (result i32)
    (local i32)
    local.get 0
    if  ;; label = @1
      i32.const 0
      local.set 1
      loop  ;; label = @2
        local.get 1
        i32.const 3
        i32.gt_u
        if  ;; label = @3
          local.get 1
          return
        end
        local.get 1
        i32.const 1
        i32.add
        local.tee 1
        local.get 0
        i32.lt_u
        br_if 0 (;@2;)
      end
    end
    local.get 1)
  (memory (;0;) 1)
  (export "mem" (memory 0))
  (export "f" (func 0)))
```
gasm executes the following opcodes:
```
op=0x20
op=0x4
op=0x41
op=0x21
op=0x3
op=0x20
op=0x41
op=0x4b
op=0x4 // if
op=0x0 // <- this is wrong
```

See the example code (and the binary module min.wasm):
https://github.com/ktye/goissues/blob/master/gasm/7/main.go

**Comment by AdwindOne** [comment]
I have the same question    	OptCodeUnreachable  OptCode = 0x00    it same as  index out of range -1    

a+b >9 9999 9999   can lead to this question

**Comment by mathetake** [comment]
resolved by #27
